lib/systemd/system-sleep/pop-default-settings_bluetooth-suspend

```diff
@@ -2,15 +2,19 @@
 
 set -e
 
-#TODO: save rfkill state?
 case "$2" in
     suspend | hybrid-sleep)
         case "$1" in
             pre)
-				rfkill block bluetooth
+                if [ $(cat /var/lib/systemd/rfkill/*bluetooth) -eq 1 ]; then
```

Comment:
What if there is more than one `bluetooth` device?  On my system I have 3 entries:

```
$ ls  /var/lib/systemd/rfkill/*bluetooth
/var/lib/systemd/rfkill/pci-0000:00:14.0-usb-0:7:1.0:bluetooth
/var/lib/systemd/rfkill/pci-0000:00:14.0-usb-0:7:bluetooth
/var/lib/systemd/rfkill/platform-dell-laptop:bluetooth
```

Therefore, the `if` statement fails because it returns too many values:

```
$ [ $(cat /var/lib/systemd/rfkill/*bluetooth) -eq 0 ]
-bash: [: too many arguments
```

A possible workaround is to sort and take the highest one:

```
[ $(cat /var/lib/systemd/rfkill/*bluetooth | sort -r | head -n 1) -eq 0 ]
```

This will guarantee that you only get one value (and the highest one).

Reply:
I check this with a second bluetooth adapter. I see why this is an issue. This seems like a good fix to me.